src/components/Header/index.scss

```diff
@@ -27,6 +27,8 @@ $Second: #5bc0be;
 		font: 26px/42px Hind Siliguri;
```

Comment:
Ist in der Main CSS definiert

Reply:
hab ich nur übernommen von der original CSS datei